Makefile

```diff
@@ -85,4 +85,4 @@ docker-push:
 # Marking this as PHONY so it will build every time (so you always have the correct version locally)
 .PHONY: $(CONTROLLER_GEN)
 $(CONTROLLER_GEN):
-	go build -o $(CONTROLLER_GEN) sigs.k8s.io/controller-tools/cmd/controller-gen
+	cd hack/tools && go build -o $(CONTROLLER_GEN) sigs.k8s.io/controller-tools/cmd/controller-gen
```

Comment:
Wouldn't this give us a potential diff between the vendored version of controller-tools for project and for the controller generation here?